ros2_control_demo_robot/controllers/rrbot_forward_controller_position_sensor.yaml

```
@@ -0,0 +1,33 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 2  # Hz
+
+    forward_command_controller_position:
+      type: forward_command_controller/ForwardCommandController
+
+    joint_state_controller:
+      type: joint_state_controller/JointStateController
+
+    ur_controllers:
+      type: controller_plugins/ForceTorqueStateController
+
+forward_command_controller_position:
+  ros__parameters:
+    joints:
+      - joint1
+      - joint2
+    interface_name: position
+
+ur_controllers:
+  ros__parameters:
+    sensor_name: tcp_fts_sensor
+    state_interface_names:
+      - fx
+      - fy
+      - fz
+      - tx
+      - ty
+      - tz
+    frame_id: kuka_tcp
```

```suggestion
    frame_id: rrbot_tcp
```

This change comes in ros-controls/roadmap#32